src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -168,9 +168,11 @@ class EmojiPickerMenu extends Component {
                 return;
             }
 
-            // Return if the key is related to any tab cycling event so that the default logic
-            // can be executed.
-            if (keyBoardEvent.key === 'Tab' || keyBoardEvent.key === 'Shift' || keyBoardEvent.key === 'Enter') {
+            // Enable keyboard movement if tab or enter is pressed or if shift is pressed while the input
+            // is not focused, so that the navigation and tab cycling can be done using the keyboard without
+            // interfering with the input behaviour.
```

Comment:
Is this good @mountiny?

Reply:
looks good to me, thanks @Santhosh-Sellavel and @BeeMargarida 🙇